### ISSUE TITLE
Qt5Agg backend not rendering nicely on Windows HiDPI display

### ISSUE THREAD

**Issue by rgbrown** [matplotlib/matplotlib#7813]
To help us understand and resolve your issue, please fill out the form to the best of your ability. You can feel free to delete the sections that do not apply.

### Bug report

**Bug summary**

Figures rendered on my Surface Pro 3 are very blurry. It looks like the figures are scaled to compensate for my device's DPI scaling (natively 1.5x). This hadn't been the case up until recently. Disabling DPI awareness from Windows doesn't help. Using Anaconda 4.2

**Code for reproduction**

```python
import matplotlib
matplotlib.use('Qt5Agg')

import matplotlib.pyplot as plt
from numpy import linspace, sin

x = linspace(0, 10, 1000)
plt.plot(x, sin(x))
plt.show()
#
```

**Actual outcome**
<img width="481" alt="capture" src="https://cloud.githubusercontent.com/assets/1946616/21911700/ccbc3d50-d986-11e6-81be-c306a15e749c.PNG">

**Expected outcome**

Until recently I would just get small (no attempt to compensate for HiDPI) figures without any blur, I'm not sure what has triggered the change.

**Matplotlib version**
Matplotlib 1.5.3
Python 3.5.2
Anaconda 4.2.0
on Windows 10

**Comment by tacaswell** [comment]
This may be addressed by https://github.com/matplotlib/matplotlib/pull/7507 but is only on the 2.x and master branch (not in a tagged release).

**Comment by QuLogic** [comment]
This should be fixed by #7507 so long as you use Qt5.
